data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/bulk/JavaClientAccumulatingUncompressedBulkRequest.java

```diff
@@ -0,0 +1,96 @@
+package org.opensearch.dataprepper.plugins.sink.opensearch.bulk;
```

Comment:
This class is unchanged from the original `JavaClientAccumulatingBulkRequest` class